src/networkWrapper/protocols/abstract/realtime.js

```diff
@@ -212,7 +212,7 @@ class RTWrapper extends KuzzleEventEmitter {
         return cb(error);
       }
       this.on(response.result.channel, data => {
-        data.fromSelf = data.volatile !== undefined && data.volatile.sdkInstanceId === this.id;
+        data.fromSelf = data.volatile !== undefined && data.volatile !== null && data.volatile.sdkInstanceId === this.id;
```

Comment:
is there any chance `data.volatile`  can be a scalar?

Reply:
No, the `RequestInput` object forbids it: https://github.com/kuzzleio/kuzzle-common-objects/blob/master/lib/models/requestInput.js#L249

Reply:
ok, great